test/base/InterestConsensusProcessRequestTest.js

```diff
@@ -0,0 +1,185 @@
+const withData = require('leche').withData;
+const { t, getLatestTimestamp, THIRTY_DAYS, ONE_DAY } = require('../utils/consts');
+const { createResponseSig, hashRequest } = require('../utils/hashes');
+const ethUtil = require('ethereumjs-util')
+const { interestConsensus } = require('../utils/events');
+
+// Smart contracts
+const InterestConsensus = artifacts.require("./mock/base/InterestConsensus.sol");
+
+
+contract('InterestConsensusProcessRequestTest', function (accounts) {
+    let instance
+    const lendersContract = accounts[1]
+    const nodeOne = accounts[2]
+    const nodeTwo = accounts[3]
+    const nodeThree = accounts[4]
+    const nodeFour = accounts[5]
+    const lender = accounts[9]
+    const endTime = 34567
+
+    let currentTime
+
+    const interestRequest = {
```

Comment:
I think it can be moved to as helper function. it will be used in other unit tests and integration tests.

Reply:
done